pkgs/by-name/dm/dmenu-rs-full/package.nix

```
@@ -0,0 +1,92 @@
+{
+  stdenv,
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+  aspell,
+  cargo,
+  expat,
+  fontconfig,
+  libXft,
+  libXinerama,
+  m4,
+  pkg-config,
+  python3,
+  xclip,
+  xdg-utils,
+}:
+
+# This version of the dmenu-rs package is built with all the plugins that have
+# been checked into the upstream repository. If you'd like to further
+# customize dmenu-rs, either disabling specific plugins or enabling additional
+# plugins from outside of the dmenu-rs repository, you'll have to build it
+# from the source.
+# See: https://github.com/Shizcow/dmenu-rs#plugins
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation rec {
```
Any reason to not use `rustPlatform.buildRustPackage` ?